.gitignore

```diff
@@ -8,3 +8,4 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+.idea
```

Comment:
This should have its own comment, or at least be separated by an empty line for clarity.

Reply:
So, I already squashed several commits, sorry for this

Reply:
Not talking about *commits* here, but about comments. This looks like it still belongs to the

```ignore
# These are backup files generated by rustfmt
```

above.

Reply:
It does seem like it would also belong into its own commit, but I don't think that's a big deal in this case.